tests/Http/PostTest.php

```diff
@@ -776,26 +761,20 @@ public function testPostsIndexAsAdminHiddenPosts()
      */
     public function testPostsIndexAsOwner()
     {
-        // Owners should only see accepted posts and their own pending/rejected posts.
-        $posts = factory(Post::class, 2)->create();
-        $rejectedPosts = factory(Post::class, 'rejected', 3)->create();
+        $userId = $this->faker->unique()->northstar_id;
+        factory(Post::class, 2)->create(['northstar_id' => $userId]);
+        factory(Post::class, 'rejected', 1)->create(['northstar_id' => $userId]);
 
-        $northstarId = $this->faker->northstar_id;
-
-        foreach ($posts as $post) {
-            $post->northstar_id = $northstarId;
-            $post->save();
-        }
+        $otherId = $this->faker->unique()->northstar_id;
+        factory(Post::class, 'rejected', 4)->create(['northstar_id' => $otherId]);
 
-        foreach ($rejectedPosts as $rejectedPost) {
-            $rejectedPost->northstar_id = $this->faker->unique()->northstar_id;
-            $rejectedPost->save();
-        }
 
-        $response = $this->withAccessToken($northstarId)->getJson('api/v3/posts');
-        $response->assertStatus(200);
-        $response->assertJsonCount(2, 'data');
+        // Owners should only see their own pending/accepted posts, and not
```

Comment:
it might be more clear to say "Owners should be able to see their own posts of any status, but not pending or rejected posts from other users" since we are showing this user their own post which has been rejected

Reply:
That reads a lot better! Updated in 4227bd7. 👍